server/prepare.c

```diff
@@ -303,6 +303,7 @@ static void send_capture_info(
     format_timestamp_message(
         timestamp_message, MAX_RESULT_LENGTH, pcap_start_tsp);
     format_attribute(&element, "start_time", "%s", timestamp_message);
+    format_attribute(&element, "hw_time_offset", "%"PRIi64, pcap_hw_ts_offset);
```

Comment:
I notice this is in nanoseconds, maybe we should rename to `hw_time_offset_ns`?

Reply:
Yes, it sounds better